lib/tests/streamlit/elements/exception_test.py

```diff
@@ -110,6 +116,30 @@ def test_uncaught_app_exception(self):
         assert proto.message == _GENERIC_UNCAUGHT_EXCEPTION_TEXT
         assert proto.type == "AttributeError"
 
+    def test_uncaught_app_exception_none_config_setting(self):
+        """
+        Corresponds to the "none" config option.
+        """
+        err = None
+        try:
+            st.format("http://not_an_image.png", width=-1)
+        except Exception as e:
+            err = UncaughtAppException(
+                e, show_message=False, show_trace=False, show_type=False
+            )
+        self.assertIsNotNone(err)
+
+        # Marshall it.
+        proto = ExceptionProto()
+        exception.marshall(proto, err)
+
+        for line in proto.stack_trace:
+            # assert message that could contain secret information in the stack trace
```

Comment:
Is this comment correct? It sounds like we check that the secret line is in the original stack trace (but not in the proto stack trace?) I think to avoid false positives, it should also go through the original stack trace and do the positive assertion. For example, if we have a typo in the assertion string or it changes, it will pass forever (because `not in line` is always true), but would lose its purpose.

Reply:
Oh, thanks for catching this. I missed updating this part of the test somehow. In this test the `stack_trace` should be empty. 

I can update the original test separately as well to add the positive assertion.